### PR TITLE
fix(empty-state): moved basic example

### LINKED ISSUE #2469: Change order of empty state examples in documentation
Goal: Shift the order of the empty state examples so that the "basic empty state" comes AFTER small, large, and [the future] extra large empty state examples, but BEFORE "with primary element" variation.

The reason for this is because this "basic" empty state variation is not one that should be used in mockups/actual implementation. Therefore, as an alternative to deleting it all together (in case a use case does come up in the future), we have decided to make it less visible in the page.

@mcarrano

### PR DESCRIPTION
**Pull request by mattnolting**
fixes #2469

### CODE REVIEW

File: src/patternfly/components/EmptyState/examples/EmptyState.md [hunk 1]
```diff
@@ -5,11 +5,12 @@ cssPrefix: pf-c-empty-state
...
 ## Examples
-```hbs title=Basic
-{{#> empty-state}}
+
+```hbs title=Small
+{{#> empty-state empty-state--modifier="pf-m-sm"}}
   {{#> empty-state-icon}}{{/empty-state-icon}}
   {{#> title titleType="h1" title--modifier="pf-m-lg"}}
-    Empty state
+    Empty State
```
Comment (mcoker) [inline src/patternfly/components/EmptyState/examples/EmptyState.md]: some scope creep, can we fix this so it's sentence case?

```suggestion
    Empty state
```

**Comment by mattnolting** [comment]
@mcoker updated
